input_examples/forcings/eurec4a_20191209_12_eul.yaml

```diff
@@ -1,6 +1,7 @@
 trajectory      : eurec4a_20191209_12_eul
 domain          : eurec4a_circle_eul
 gradient_method : regression
+advection_velocity_method : mean
```

Comment:
I think I would call this `domain_mean` rather than `mean` to make it clearer what the domain is calculated over, what do you think?

Reply:
and maybe it should be `advection_velocity_sampling_method` instead? I realise it's long, but might be more helpful

Reply:
I agree "domain_mean" is more explicit, happy to change this. Not sure about the second point, sampling may give the input it is about the mask used, whereas this is more about local V global. Happy to chat about this later.

Reply:
Have implemented "advection_velocity_sampling_method" for now. Did not find a better name quickly.